tests/time_tests/c++/pipeline.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
```

Comment:
The directory structure should follow the one in other openvino modules: module/src for internals and module/include for public includes. There is no "c++" directory

Reply:
Will be fixed after completing code review

Reply:
Updated, please, review again